extension/ballastextension/README.md

```diff
@@ -6,12 +6,38 @@ Memory Ballast extension enables applications to configure memory ballast for th
 
 The following settings can be configured:
 
-- `size_mib` (default = 0, disabled): Is the memory ballast size, in MiB.
+- `size_mib` (default = 0, disabled): Is the memory ballast size, in MiB. 
+  Take higher priority than `size_in_percentage` if both enabled at the same time.
```

Comment:
Since few users read docs, should we just not allow setting both flags at the same time?  That would be a clearer signal that they aren't compatible.